docs/validator/staking-overview.md

```diff
@@ -18,5 +22,6 @@ This section introduces you to the validation process and some additional resour
 
     * [The Beginner’s Guide to the NEAR Blockchain](https://nearprotocol.com/blog/the-beginners-guide-to-the-near-blockchain/) to get a high level overview on NEAR.
     * [Nightshade](../technical/nightshade.md) to gain a more detailed understanding of our consensus mechanism.
+    * [Economics in Sharded Blockchain](https://nearprotocol.com/wp-content/uploads/2019/10/Economics_in_Sharded_Blockchain.pdf) to know more about the incentives structure of NEAR.
```

Comment:
we have some better links for these now: 

* [The Beginner’s Guide to the NEAR Blockchain](https://nearprotocol.com/blog/the-beginners-guide-to-the-near-blockchain/) to get a high level overview on NEAR.
* [The NEAR Whitepaper](https://nearprotocol.com/papers/the-official-near-white-paper) to see the big picture
* [Economics in Sharded Blockchain](https://nearprotocol.com/papers/economics-in-sharded-blockchain/) to know more about the incentives structure of NEAR.
* [Sharding Design: Nightshade](https://nearprotocol.com/papers/nightshade/) to gain a more detailed understanding of our consensus mechanism.

Reply:
the links i've added here point to the SEO friendly URLs, btw